.changeset/tough-buses-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus": patch
+---
+
+Fix function signature of getUserInputFromProps for radio component
```

Comment:
**nit**: s/component/widget/ - in Perseus, widgets are a special thing that includes a React component, but conforms to a specific API. 
```suggestion
Fix function signature of getUserInputFromProps for radio widget
```